### DISCUSSION TITLE
Does teleport blocks resources when there's security patch ?

### DISCUSSION THREAD

**Discussion by alirezarpi** [gravitational/teleport#28675] [General]
Hi,

I had an old teleport-cluster deployed on k8s which was working that time (ver 12.5.xx) and since then I didn't touch it and didn't used the resources until now that I got the security message on login.
DB resource (which was working before) didn't work anymore and it didn't even hit the database it got blocked inside teleport-cluster, so I updated to 13.xx.xx which doesn't have security patch and it works now.

Is it intentional that teleport did that or it's just random ?
If intentional can it be disabled ?

Thank you in-advance

**Comment by webvictim** [comment]
Teleport doesn't intentionally disable anything when there's a security patch available - it just displays a warning. I think this was a coincidence!
